model/src/test/kotlin/IdentifierTest.kt

```diff
@@ -102,6 +105,45 @@ class IdentifierTest : WordSpec({
         }
     }
 
+    "Purl representations" should {
```

Comment:
Similar to last commit, could be `"toPurl()" should {`.

Reply:
See above.